app/views/govuk_wrapper.scala.html

```diff
@@ -86,6 +87,9 @@
     ssoUrl = None,
     scriptElem = Some(scripts),
     gaCalls = None)
+    <!-- Google Tag Manager (noscript) -->
+    <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-P39VTVL" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
```

Comment:
The GTM id is hardcodes in 2 places. It might be better to keep one version of it in the config.

Reply:
Yeah I set it up like that on the covid projects originally having the same thoughts, but according to the performance analysts it won't ever change or be overridden because we should be connected to the main GTM bucket for everything now. So unless we move away from GTM it shouldn't ever need to change.

Reply:
:+1: